_posts/2017-02-28-foreman-community-newsletter---february-2017.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: Foreman Community Newsletter - February 2017
+date: 2017-02-28 12:33:42
+author: Greg Sutcliffe
+tags:
+- foreman
+- newsletter
+- community
+- fosdem
+- cfgmgmtcamp
+excerpt: |
+  A bumper roundup all the release, events, and plugin news since Christmas, as
+  well as the conference trip report!
+---
+
+Apologies for the lack of a January newsletter - the conferences in the Czech
+Republic and Belgium kept us all very busy! Here's the roundup of the last few
```

Comment:
"Czechia" is the official short name now: http://www.independent.co.uk/news/world/europe/czechia-czech-republic-new-name-english-speakers-a7323596.html :trollface: